### ISSUE TITLE
Bug: Undeclared TB_TRUECOLOR_* variables

### ISSUE THREAD

**Issue by Stef-Gijsberts** [termbox/termbox2#26]
## Description
When I try to use this library, I get compiler errors.

## Edit: Solution
I just noticed that git-bruh mentioned the following in #25:
> This also fixes compile errors when building without truecolor.

So I suspect that PR #25 will fix this issue.

## Steps to reproduce
1. Install termbox2 by first cloning the repository, then running `sudo make install` from within the repository folder.
2. Write the following program to a file called `minimal.c`:
```c
#define TB_IMPL
#include <termbox.h>

int main() {
    tb_init();
    tb_shutdown();
}
```
2. Try to compile the program using gcc:
```bash
gcc minimal.c -o minimal
```

3. Observe multiple compilation errors:
```
In file included from minimal.c:2:
/usr/local/include/termbox.h: In function ‘send_attr’:
/usr/local/include/termbox.h:2821:21: error: ‘TB_TRUECOLOR_BOLD’ undeclared (first use in this function)
 2821 |         attr_bold = TB_TRUECOLOR_BOLD;
      |                     ^~~~~~~~~~~~~~~~~
/usr/local/include/termbox.h:2821:21: note: each undeclared identifier is reported only once for each function it appears in
/usr/local/include/termbox.h:2822:22: error: ‘TB_TRUECOLOR_BLINK’ undeclared (first use in this function)
 2822 |         attr_blink = TB_TRUECOLOR_BLINK;
      |                      ^~~~~~~~~~~~~~~~~~
/usr/local/include/termbox.h:2823:23: error: ‘TB_TRUECOLOR_ITALIC’ undeclared (first use in this function)
 2823 |         attr_italic = TB_TRUECOLOR_ITALIC;
      |                       ^~~~~~~~~~~~~~~~~~~
/usr/local/include/termbox.h:2824:26: error: ‘TB_TRUECOLOR_UNDERLINE’ undeclared (first use in this function); did you mean ‘TB_CAP_UNDERLINE’?
 2824 |         attr_underline = TB_TRUECOLOR_UNDERLINE;
      |                          ^~~~~~~~~~~~~~~~~~~~~~
      |                          TB_CAP_UNDERLINE
/usr/local/include/termbox.h:2825:24: error: ‘TB_TRUECOLOR_REVERSE’ undeclared (first use in this function)
 2825 |         attr_reverse = TB_TRUECOLOR_REVERSE;
      |                        ^~~~~~~~~~~~~~~~~~~~
```

**Comment by adsr** [comment]
Thanks for reporting. Should be fixed by 349db40e96464fa0bc6251903c9eaa118bfd8c8a.

**Comment by adsr** [comment]
Filed https://github.com/termbox/termbox2/issues/27 to get the tests running for non-truecolor builds, which would catch this sort of thing.
